data/human/human missions.txt

```diff
@@ -4606,7 +4606,6 @@ mission "Hai Wormhole Warning"
 			has "First Contact: Unfettered: declined"
 		not "Block Hai Wormhole Warning: offered"
 		not "Wanderers: Alpha Surveillance E: offered"
```

Comment:
The mission by this name doesn't exist anymore, but there's still a point in HR after which this mission should no longer offer.
```suggestion
		not "Wanderers: Alpha Surveillance E: offered"
		not "hr: meet the team"
```